app/models/article.rb

```diff
@@ -352,22 +350,6 @@ def update_score
                    spaminess_rating: BlackBox.calculate_spaminess(self))
   end
 
-  def liquid_tags_used(section = nil)
-    content =
-      case section
-      when :body
-        body_markdown
-      when :comments
-        comments_blob
-      else
-        "#{body_markdown}#{comments_blob}"
-      end
-
-    MarkdownParser.new(content, source: self, user: user).tags_used
-  rescue StandardError
-    []
-  end
-
```

Comment:
This was originally implemented _just_ for the validation of the `PollTag`. Now that we handle the validation differently, we don't need this on the model anymore.

Reply:
I LOVE that this is not longer on the article!!!